.github/workflows/publish-cdn.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Publish: Publish to CDN'
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'The repository version to publish'
+        required: true
+        type: string
+
+  # This workflow is intended to be called from the primary publish workflow
+  # automatically. The manual trigger is maintained for partially failed workflows
+  # or edge cases that require this be run independently.
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The repository version to publish'
+        required: true
+        type: string
+
+env:
+  GH_ACTION_ROLE: arn:aws:iam::073865013699:role/CDN_Autocomplete_UI_GHA_Role
+  AWS_REGION : us-east-1
+
+jobs:
+  publish-to-cdn:
+    name: Publish to CDN
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.x
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build compiled package for publishing
+        uses: "./.github/actions/compile"
+        with: 
+          version: ${{ inputs.version }}
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@010d0da01d0b5a38af31e9c3470dbfdabdecca3a # v4.0.1
+        with:
+          role-session-name: cdn-publish-action
+          role-to-assume: ${{ env.GH_ACTION_ROLE }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Copy files to CDN
+        run: |
+          cp dist/constructorio-ui-autocomplete-bundled dist/${{ inputs.version }}.js
```

Comment:
Does the first argument need `.js` extension? How does it work without the extension 🤔

Reply:
It does! My bad - forgot to add it.

Fixed in 10e4f3e6fb7a1555e27ea7869418f18547be0951.